pkg/volume/gcepd/attacher.go

```diff
@@ -173,6 +173,8 @@ func (attacher *gcePersistentDiskAttacher) WaitForAttach(spec *volume.Spec, devi
 				// A device path has successfully been created for the PD
 				glog.Infof("Successfully found attached GCE PD %q.", pdName)
 				return path, nil
+			} else {
+				glog.V(4).Infof("could not verify GCE PD (%q) is attached, device path is empty", pdName)
```

Comment:
hm I wonder if every second may get too verbose.  Could we add some throttling to this so we only log once every 10 seconds?

Reply:
Having it at log level 4 should be enough IMO. When I set log levels higher for debugging I'd like to see the log exactly when the error occurs